pages/about/index.md

```diff
@@ -28,10 +49,69 @@ Would you like to get on the list and support us financially? Check out the [spo
+- [**Slack**](https://slack.com/signin#/signin): Business Communication Platform that permits to operate with multiple channels (public and private). It currently has more than 10 million daily active users worldwide. It was recently acquired by Salesforce.
+- [**Adidas**](https://www.adidas.com/us): Company dedicated to the manufacture of sports equipment and fashion products. It is the largest sportswear manufacturer in Europe, second largest in the world. This company has more than 150.000 sales outlets around the world.
+- [**Salesforce**](https://login.salesforce.com/?locale=es): Business consultancy company. It provides (CRM) service and other services. It manages from small businesses to FORTUNE 500 companies and it has more than 150,000 customers.
+- [**IBM**](https://www.ibm.com): Company that manufactures and markets hardware and software. It has operations in over 170 countries and provides hosting and consulting services in many areas.
+- [**SAP**](https://www.sap.com/index.html): Company dedicated to the design of computer products for business management. Develops business software to manage operations and business-to-customer relationships. It’s a large company with 100,330 employees.
+- [**IQVIA**](https://www.iqvia.com/"): Company providing services for the combined health information technology and clinical research industries. It employs more than 58.000 people in over 100 countries
 
-This project started at Hitch (a.k.a. API Changelog). We are in love with API machine-readable documentation formats (OpenAPI, RAML, etc.) and always encouraged our customers to use a machine-readable definition for their APIs because it allows them to get extra benefits. From documentation and code generation to setup their own [API Assistant](http://apiassistant.com).
+## Open governance model
 
-Internally, we used a message-driven microservices architecture and we couldn't have all those tools that we have with the HTTP APIs. All those things you get because you have your API defined with a machine-readable documentation format. And, on top of that, we noticed that some of our customers were creating IoT APIs over MQTT and they couldn't get all the benefits from Hitch because the existing specifications don't support message-driven APIs.
+Free software allows users to apply, modify, and continue in the chain by distributing copies to help others and, in this case, disseminating modified versions of the specification. 
 
-**It was time to come up with something good enough for everyone instead of having every company doing its own battle. We can learn a lot together if we join forces, so we decided to create AsyncAPI!**
+The partnership mentioned above with Postman speeded up the initiative's work in transferring AsyncAPI to a foundation and forming an open governance model to assure the community that a single company does not control AsyncAPI Initiative. This move takes the project to a higher level. 
```

Comment:
we no longer talk about this partnership as it was before, so nothing is really mentioned above.